src/styles/index.scss

```diff
@@ -5,6 +5,13 @@
 @use 'homePopularProductCard';
 @use 'homeCategoryProductCard';
 @use 'homeTopSeller';
+@use "login";
+@use "authBlueSide";
+@use "loader";
 @import 'tailwindcss/base';
 @import 'tailwindcss/components';
 @import 'tailwindcss/utilities';
+*{
+    font-family: 'Poppins', sans-serif !important;
```

Comment:
[Here](https://github.com/atlp-rwanda/e-comm-team-techsmith-fn/blob/ft-login-page-185218916/src/styles/base.scss#L15) is another reference of this line.